### DISCUSSION TITLE
using babel-polyfill by cdn

### DISCUSSION THREAD

**Discussion by m3rri** [babel/babel#13365] [Q&A] [answered]
my project built by Spring not node.js, so I'm going to apply babel-standalone and babel-polyfill using cdn.
the document says "babel-polyfill" has been deprecated, but I wonder if i can use this.
it's a first time using it at work, so I want to make it sure..

▼code
`<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/babel-standalone/6.26.0/babel.min.js"></script>
<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/7.12.1/polyfill.min.js"></script>`

**Comment by m3rri** [comment, marked as answer]
i think it works well.

`<script type="text/javascript" src="https://unpkg.com/core-js-bundle@3.12.1/minified.js"></script>
<script type="text/javascript" src="https://unpkg.com/regenerator-runtime@0.13.7/runtime.js"></script>
<script type="text/javascript" src="https://unpkg.com/@babel/standalone@7.14.3/babel.min.js"></script>`

**Reply by nicolo-ribaudo** [reply]
Yes, this is the correct way of doing it!

**Reply by m3rri** [reply]
wow.. Thank you 😂
